File: src/org/r3/db/project/siteresource/SiteResourceDTO.java
//
// GPA Gruppo Progetti Avanzati s.r.l. - 
// Progetto UNEP 2012 per PosteItaliane S.p.a.
//
//
// TODO: Readme Section.
//


package org.r3.db.project.siteresource;

public class SiteResourceDTO extends SiteResourceDTOBase
{
	public enum ResourceType {
		actor
	}
	
    public SiteResourceDTO()
    {
        super();
    }

    public SiteResourceDTO(SiteResourceDTO another)
    {
        super(another);
    }

}

File: WebContent/org/r3/db/project/siteresource/SiteResourceDTO.js
//
// GPA Gruppo Progetti Avanzati s.r.l. - 
// Progetto UNEP 2012 per PosteItaliane S.p.a.
//
//
// TODO: Readme Section.
//



jQuery.r3Utils.namespace("org.r3.db.project.siteresource");

org.r3.db.project.siteresource.SiteResourceDTO = Class.extend(
{
	 init: function(aDataBean)
	 {  
	 	 	 this.dtrowclass   = "grade_siteresource";
       this.dtokey       = aDataBean.dtokey   || null;
       this.dtodescr     = aDataBean.dtodescr || null;
       
        this.site = aDataBean.site || null;
        this.resourceid = aDataBean.resourceid || null;
        this.profileid = aDataBean.profileid || null;
        this.evalueunit = aDataBean.evalueunit || null;
        this.price = aDataBean.price || 0.0;
        this.cost = aDataBean.cost || 0.0;
        this.recstatus = aDataBean.recstatus || null;
        this.rectimestamp = aDataBean.rectimestamp || null;
 
	 }
    
  ,compareTo: function(anItemId)
   {
   	   if (!this.dtokey)
   	       return -99;
   	       
   	   if (this.dtokey < anItemId)
   	       return -1;
   	   else if (this.dtokey > anItemId)
   	   	   return  1;
   	   	   
   	   return 0;
   }
   
});
